lib/src/Uncertainty/Model/EllipticalDistribution.cxx

```diff
@@ -406,6 +406,21 @@ Point EllipticalDistribution::computePDFGradient(const Point & point) const
     // dPDF / dsigma_i
     pdfGradient[dimension + i] = pdfGradient[i] * u[i] - normalizationFactor_ * phi * iSigma;
   }
+  if (getDimension() > 1)
```

Comment:
No hope to derive an analytical expression of it? E.g. using Jacobi's identity https://en.wikipedia.org/wiki/Jacobi%27s_formula for the ugly determinant part?

Reply:
still have to compute the inverse of the gradient, it's not obvious that it will become cheaper, deferring